src/tmpfiles/tmpfiles.c

```diff
@@ -1272,8 +1272,10 @@ static int write_one_file(Item *i, const char *path) {
 
         bn = basename(path);
 
+        flags = O_NONBLOCK|O_CLOEXEC|O_WRONLY|O_NOCTTY;
+
         /* Follows symlinks */
-        fd = openat(dir_fd, bn, O_NONBLOCK|O_CLOEXEC|O_WRONLY|O_NOCTTY, i->mode);
+        fd = openat(dir_fd, bn, i->force ? flags|O_APPEND : flags, i->mode);
```

Comment:
Hmm, that's just too confusing. Please add an earlier commit which renames `force` to something else, e.g. `append_or_force` or some better name.

Reply:
Hmm..I see your point in this instance but I worry that renaming this will make other cases for `+` / `force` more confusing. How would you feel about creating a new modifier for this that is clearly for "append" - perhaps `w>` or maybe using a letter like `wa` and then creating a new field in the `Item` struct like `bool append:1;` This would make the line in question much clearer:

```C
i->append ? flags|O_APPEND : flags
```

Something like

```
w> /tmp/13291.out - - - - some line
```

I could imagine a situation where a user might want to prepend a file so we could have `w<` or `wp`?

`<` and `>` have meaning in the shell that it "overwrites" which so I could see an argument against those so certainly open to other symbol suggestions.

Reply:
We already use + for appending: a+, A+. So I don't think we should use a different character.

Reply:
Good point. Changed with https://github.com/systemd/systemd/pull/13709/commits/bf9780228e7e83e25eb30ca56269b6e7e51c41b3

Reply:
Actually, I think I screwed up the `rebase` - I'll have to fix it later today. :disappointed:

Reply:
Fixed